frontend/src/components/Sidebar/__snapshots__/Sidebaritem.SublistExpanded.stories.storyshot

```diff
@@ -50,10 +50,10 @@
                   class="MuiList-root css-2l483y-MuiList-root"
                 >
                   <li
-                    class="css-b4fdej"
+                    class="css-gcxs36"
                   >
                     <a
-                      class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-padding MuiListItem-button css-sayhe9-MuiButtonBase-root-MuiListItem-root"
+                      class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-padding MuiListItem-button Mui-selected css-sayhe9-MuiButtonBase-root-MuiListItem-root"
```

Comment:
Can you please explain why this test result changed? I don't see any mention of why, so I'm wondering if it's intentional or a bug?